### ISSUE TITLE
Docker image error

### ISSUE THREAD

**Issue by sixpenrose** [ElementsProject/lightning-charge#77]
Hey, 
I'm running into an error when running the docker container. The command to run the container hangs up on trying to start the lightningd. From the `docker-entry.sh`, it seems to be around this line: https://github.com/ElementsProject/lightning-charge/blob/master/bin/docker-entrypoint.sh#L71

```
docker run -it -u `id -u` -v `pwd`/data:/data -p 9735:9735 -p 9112:9112 shesek/lightning-charge --api-token mySecretToken
Starting bitcoind... waiting for cookie... waiting for RPC... ready.
Starting lightningd... waiting for RPC unix socket... 
```

I tried removing the downloaded image and re-downloaded - to make sure it's not a cache issue. 

Any suggestions? 

Thank you,

**Comment by shesek** [comment]
Thanks for the report! This should be fixed in v0.4.22. Can you try and see if that works?

**Comment by sixpenrose** [comment]
@shesek 
That was quick, and worked like a charm! Thanks for the commit and work!
